### ISSUE TITLE
MakerDAO requesting a lot of göEth

### ISSUE THREAD

**Issue by ultraschuppi** [goerli/testnet#87]
Hi!

the [Protocol Engineering](https://forum.makerdao.com/tag/pe-001) team at MakerDAO made their call to move out of Kovan and into Görli as their testing environment a couple of weeks ago. 

I reached out on their behalf to María Paula to get some precious ETH and she [transfered 20k](https://goerli.etherscan.io/tx/0x48545512c27a4ef8738b2a5cebd6a0878dcaf7834914df9e0ddebc31439a4365) which will be enough to deploy the whole stack of smart contracts into görli :rocket:

For those not familiar with Maker: users can add collateral (e.g. ETH) and mint DAI (a stablecoin softpegged to the USdollar) from that. 

To get the test instance on Görli into a prod-like scenario, it would be extremely helpful to be able to deposit similar amounts of göETH as we have on the mainnet. Thats a lot - just on ETH-A - which is the biggest ETH-based vault-type on mainnet, [2 MM of ETH are currently locked there](https://daistats.com/#/collateral).

![image](https://user-images.githubusercontent.com/6298245/125833641-227926b6-d800-47e3-baa8-e943e56a78e7.png)

The alternative would be to come up with some fake token, but this just leads to a an unnecessary difference between mainnet and Görli.

So, TL;DR: I would like to ask if the Protocol Engineering team can get a big chunk of göETH.

Thanks and cheers
 [tim](https://forum.makerdao.com/u/ultraschuppi/summary)

**Comment by roninkaizen** [comment]
please pm in gitter or discord
if you can
@ultraschuppi

**Comment by q9f** [comment]
Please message me a goerli address and desired amount at goerli@fault.dev if you need more!
